CMakeLists.txt

```diff
@@ -370,8 +370,15 @@ install(FILES "${CMAKE_CURRENT_BINARY_DIR}/DirectXTex.pc"
         DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
 
 #--- Command-line tools
-if(BUILD_TOOLS AND BUILD_DX11 AND WIN32)
-  set(TOOL_EXES texassemble texconv texdiag)
+set(TOOL_EXES "")
+if(BUILD_TOOLS AND WIN32)
+  add_executable(texdiag
```

Comment:
I was using alphabetical order here: texassemble, texconv, texdiag. Please move this back down.